.github/workflows/deploy.yml

```diff
@@ -47,10 +47,10 @@ jobs:
           # bundle exec rubocop
           bundle exec rspec
           echo $?
-      # - name: Publish code coverage
-      #   uses: paambaati/codeclimate-action@v2.7.5
-      #   env:
-      #     CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+      - name: Publish code coverage
+        uses: paambaati/codeclimate-action@v2.7.5
```

Comment:
We could probably upgrade the version of the action - it looks like it's up to 5.0.0 now

Reply:
Done, thanks Mike for the suggestion. I guess we need to update version for Bracco, poodle too.